client/verta/verta/endpoint/_endpoint.py

```diff
@@ -202,7 +201,7 @@ def _get_by_id(cls, conn, conf, workspace, id):
 
     def _get_info_list_by_id(self):
```

Comment:
I searched the codebase for `\._get_info_list_by_id` and it's only being used in [`_cli/deployment/list.py`](https://github.com/VertaAI/modeldb/pull/3711/files#diff-66d3129bfe24e0be3417782f5ef46c1b5fcae107844de927a6fe58adaa92b4a0).